luigi/s3.py

```diff
@@ -284,23 +283,82 @@ def get_as_string(self, s3_path):
     def copy(self, source_path, destination_path, **kwargs):
         """
         Copy an object from one S3 location to another.
-
         :param kwargs: Keyword arguments are passed to the boto function `copy_key`
         """
         (src_bucket, src_key) = self._path_to_bucket_and_key(source_path)
         (dst_bucket, dst_key) = self._path_to_bucket_and_key(destination_path)
 
         s3_bucket = self.s3.get_bucket(dst_bucket, validate=True)
+        source_bucket = self.s3.get_bucket(src_bucket, validate=True)
+
+        multipart_size = 67108864
```

Comment:
Yes, this irks me too - this magic constant is present in the `put_multipart` method, and I just copied it from there. I have no idea why this was chosen as the `multipart_size` for that, either.